File: src/main/java/com/github/theword/queqiao/payload/modle/CommonHoverEvent.java
package com.github.theword.queqiao.payload.modle;


import com.google.gson.annotations.SerializedName;
import lombok.Data;

import java.util.List;

@Data
public class CommonHoverEvent {
    private String action;
    @SerializedName("base_component_list")
    private List<CommonBaseComponent> baseComponentList;
    private CommonHoverItem item;
    private CommonHoverEntity entity;
}

File: src/main/java/com/github/theword/queqiao/tool/payload/modle/hover/CommonHoverEvent.kt
package com.github.theword.queqiao.tool.payload.modle.hover;


import com.github.theword.queqiao.tool.payload.modle.component.CommonBaseComponent;
import lombok.Data;

import java.util.List;

@Data
public class CommonHoverEvent {
    private String action;
    private List<CommonBaseComponent> text;
    private CommonHoverItem item;
    private CommonHoverEntity entity;
}
